src/components/Disturber.tsx

```diff
@@ -83,9 +84,11 @@ export const Disturber = ({ navigation, publicJsonFile }: Props) => {
       <View style={[styles.containerRadius, { backgroundColor: closestItem.backgroundColor }]}>
         {!!closestItem && (
           <>
-            <TouchableOpacity onPress={setDisturberComplete} style={styles.closeButton}>
-              <Icon.Close color={colors.darkText} size={normalize(16)} />
-            </TouchableOpacity>
+            {closestItem.showButtonToClose && (
```

Comment:
can be undefined here, right? i suggest a `true` fallback of the value, which can be overridden with `false` per configs.

Reply:
Yes, it can be It can be `undefined`. I set `true` by default. 

https://github.com/smart-village-solutions/smart-village-app-app/commit/6d16fa29134aa86fad0d96aab7a2c727e19a1592